recipes/tilt/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "0.16.1" %}
+{% set commit = "1f21e296d6a1029223d273bcf43dc6b0bf71c0e8" %}
+{% set date = "2020-07-17" %}
+
+package:
+  name: tilt
+  version: {{ version }}
+
+source:
+  url: https://github.com/tilt-dev/tilt/archive/v{{ version }}.tar.gz
+  sha256: 4df87f9efaa83e518d45f0ea10b6b3f7839076ab70731e434ce2f45f794770d4
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }} 1.13
+    - clangxx_osx-64  # [osx]
+  run_constrained:
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
+
+test:
+  commands:
+    - tilt version
+
+about:
+  home: https://github.com/tilt-dev/tilt
+  license: Apache-2.0
+  license_family: Apache
```

Comment:
No need for this any more as it's added automatically based on the value of `license`:
```suggestion
```